test/test_files/tinysnb/range_literal/range_literal.test

```diff
@@ -0,0 +1,121 @@
+-GROUP TinySnbReadTest
+-DATASET CSV tinysnb
+
+--
+
+-CASE SingleUpperBound
+-STATEMENT MATCH(a:person {fName: "Alice"})-[e:knows*..2]->(b:person) RETURN e;
```

Comment:
Maybe just do a COUNT(*) here. I feel the ground truth is a bit long. Though this is up to u.